v2/cmd/asoctl/cmd/template.go

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package cmd
+
+import (
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+
+	"github.com/Azure/azure-service-operator/v2/cmd/asoctl/internal/template"
+)
+
+type templateOptions struct {
+	source  string
+	version string
+
+	raw        bool
+	crdPattern []string
+}
+
+var expectedTagRegex = regexp.MustCompile(`^v\d.\d.\d$`)
```

Comment:
Those periods will match any one character - to match a literal period, you need to quote them:

```suggestion
var expectedTagRegex = regexp.MustCompile(`^v\d\.\d\.\d$`)
```

Reply:
Ahh good catch